### DISCUSSION TITLE
Screen Mirroring fps

### DISCUSSION THREAD

**Discussion by himlastar** [BimmerGestalt/AAIdrive#833] [General]
Is there a way to get higher FPS ? I have 1-2 FPS while screen mirroring.

Using Google Pixel 9 XL, maybe some display configuration that makes android resolution lower, would that improve FPS?

**Comment by hufman** [comment]
The only way to have higher FPS is to have a higher transport speed, by trying to use USB. However, MyBMW does not reliably support USB, and USB mode doesn't support music, so this is suboptimal.
The app ignores the android resolution and only renders at the cars resolution. Unfortunately, the car image widget doesn't support any stretching, so the app has to send the full car resolution instead of a half resolution that is stretched to fill.
